news/2024-05-15-numpy-2-migration.md

```diff
@@ -0,0 +1,9 @@
+# NumPy 2 migration
+
+In preparation for the NumPy 2 release, we have begun a migration of all
+conda-forge packages that depend on `numpy` during the build (so in
+`requirements/host`). The migration status is tracked [on our status page](
+https://conda-forge.org/status/migration/numpy2 ). Particular package's status
+can be found there. Specific recommendations for maintainers updating packages
+can be found in the migration PRs themselves. Please reach out through the
+usual channels if you have any questions.
```

Comment:
Do we want a centralized issue on NumPy 2 bringup for package maintainers?

Asking as we did this with CUDA 12 and that was pretty helpful. Though other approaches can work as well